lib/CMakeLists.txt

```diff
@@ -27,17 +27,66 @@ if(USE_SIFT_GPU)
     add_definitions(-DUSING_GPU)
 endif()
 
+############## LIBELAS ##############
+
+# use sse3 instruction set
+set (CMAKE_CXX_FLAGS "-msse3")
+
+if (UNIX)
+  find_package(OpenMP)
+endif()
+
+if(OPENMP_FOUND)
```

Comment:
Seems like in OpenMP they are not following that scheme for some reason: http://www.cmake.org/cmake/help/v3.0/module/FindOpenMP.html